### PR TITLE
Feat: Error handling upon invalid input

### PR DESCRIPTION
**Pull request by Prashanth-Chandra**
Added features as mentioned in #14

### CODE REVIEW

**Comment by Prashanth-Chandra** [comment]
Added option to change color of the board. Colors are inspired from [chess.com's](https://www.chess.com/article/view/what-your-chess-board-theme-says-about-you) collection of boards (Green, Brown, Blue, Bubblegum)

File: src/getTheme.js [hunk 1]
```diff
@@ -9,3 +9,15 @@ selectedTheme.onchange = () => {
     currentArrangement = getCurrentFen();
     generateBoard(currentArrangement);
 }
+
```
Comment (SrNightmare09) [inline src/getTheme.js:12]: please make the feature to change the board theme a separate pr. thank you!

**Review by SrNightmare09 — COMMENTED** [review]
please make some minor changes mentioned

**Comment by SrNightmare09** [comment]
looks good1 merged!

**Pull request merged**